_pages/ingest/mlcp.md

```diff
@@ -49,7 +56,19 @@ This is how you would run a flow named "My Awesome Flow" for the entity named "Y
 
 ... \
 
--transform_module "/com.marklogic.hub/mlcp-flow-transform.xqy" \
+-transform_module "/MarklOgic/data-hub-framework/transforms/mlcp-flow-transform.xqy" \
 -transform_namespace "http://marklogic.com/data-hub/mlcp-flow-transform" \
 -transform_param "entity-name=YourEntityName,flow-name=My%20Awesome%20Flow,job-id=someString,options={'your':'options'}"
 </pre>
+
```

Comment:
There seems to be extra newlines here in a few places here. It needs to come out of the `<pre` segment because it'll make it not look like a cmd line. (Edit: I see the xqy example looks like that, too, so you're not first offender, but let's fix them both.)